radio/views/feedback.py

```diff
@@ -0,0 +1,16 @@
+from aiohttp import web
+import radio.services.mail
+
+
+class Collection(web.View):
+    """
+    API view that is used for email feedback post handling
+    """
+
+    async def post(self):
+        data = await self.request.post()
+        mail_data = {'sender': data['sender_email'], 'body': data['body']}
```

Comment:
Is it possible if data['body'] is empty or None? Should this feedback be saved, or just skipped?